scripts/verify_images.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Verify that all image references embedded in the release data are
+# for valid registries.
+
+verbose=false
+if [ "$1" = "-v" ]; then
+    verbose=true
+fi
+
+function debug() {
+    if ! $verbose; then
+        return
+    fi
+    echo "$*"
+}
+
+approved=true
+./pkg/release/get.sh images all | while read source_file image; do
```

Comment:
Can we change this from relative to $ROOTDIR?

ROOTDIR=$(git rev-parse --show-toplevel)

Reply:
Done